synapse/lib/cell.py

```diff
@@ -1715,6 +1715,27 @@ async def delDriveInfo(self, iden):
     async def setDriveInfoPerm(self, iden, perm):
         return self.drive.setItemPerm(iden, perm)
 
+    @s_nexus.Pusher.onPushAuto('drive:data:path:set')
+    async def setDriveItemProp(self, iden, path, valu):
```

Comment:
would we want the ability to update the version with this call?

Reply:
I'd say yea. Otherwise you have to do another kinda awkward call to `setItemData` since there's not really a way to set just the version info. I'll add that param :+1: